### DISCUSSION TITLE
score_cutoff not working in extract

### DISCUSSION THREAD

**Discussion by SnoozeFreddo** [rapidfuzz/RapidFuzz#161] [Q&A] [answered]
Hi,

i am passing 100k elements to extract() but i am only getting 5 back. my score_cutoff is 0-10 and i have no limit set.

**Comment by SnoozeFreddo** [comment]
Okay, that's interesting. If i set a limit it returns more values, if not, then i get only 5. 

I'll try to invalidate my IDE cache tomorrow. I am currently running a heavy task since 2 days and i can't restart the IDE.

**Comment by maxbachmann** [comment, marked as answer]
Limit defaults to 5. You can return all elements by using `limit=None`. This default is selected to keep a similar behavior to FuzzyWuzzy. You can find the full documentation of `process.extract` here: https://maxbachmann.github.io/RapidFuzz/process.html#extract.
